src/workbook/stakeholder-sheet/holdings-columns.ts

```diff
@@ -282,13 +282,22 @@ export class WarrantColumn {
     const myColumn = this.parent.createNestedRange({
       orientation: "top-to-bottom",
     });
+    let targetClass = stockClass;
```

Comment:
The only thing the warrant column (or non-plan awards column) would check, is to ensure that it displays the correct header for a warrant.
For example, if a warrant transaction is connected to a preferred stock class, we don't want to show that class name in the header. Instead, we want to get the resulting common class and use its name.